src/components/YourCompany/YourCompany.vue

```diff
@@ -62,8 +126,36 @@
             />
           </v-flex>
         </template>
+
+        <template v-if="isAlteration() && hasNewNr">
+          <v-flex xs3 class="mt-6">
+            <v-layout column>
+              <label><strong>Name Request Applicant</strong></label>
+            </v-layout>
+          </v-flex>
+          <v-flex xs6 class="mt-6">
+            <template v-if="hasNewNr">
+              <div>
+                <span class="font-weight-bold">Name: </span>
+                <span>{{getNameRequest.applicant.fullName}}</span>
```

Comment:
What happens if getNameRequest failed?

Reply:
Hmm, well the template is guarded by a check around the property `nrNumber`,. 
That being said, if somehow these other values are not returned from the API i could re-write these template values into a getter which will fail gracefully if the data isn't present. 
Will update

Reply:
Or add a v-if for the results?

Reply:
Oh i see. There is a v-if already around these results templates or do you mean to protect each individual display property. 
That being said, if condition was met to display these templates and somehow the data wasn't there, i think all that happens is the data isn't there.

Reply:
I didn't see if there was a v-if to verify `getNameRequest.applicant` (should be be "applicants"? lol).

I think you actually get a template error if, say, getNameRequest or getNameRequest.applicant are null.

Reply:
ah sorry, not on that property specifically. Ok, will adjust!

If 'getNameRequest`  is null.. ie the store properties it references, the template shouldn't be rendered.

Reply:
If you think it's well enough protected then leave it. I was just askin'.

Reply:
lol.. i don't think so, with regards to `applicant`.
That is the property we assign those values too, whereas `applicants` was from Namex and in the interface that gave definition to the response (obviously `applicants` being misleading because it only returns an object).

I'm not sure if it is normal practice to make an interface for expected responses but i thought what the heck in this case.

Reply:
oh no, i agree i could protect them a bit better. 
I added a getter which optional chains the Name Request before referencing applicant 👍

Reply:
> I'm not sure if it is normal practice to make an interface for expected responses but i thought what the heck in this case.

It is good practise, if not normal. Re: schema !!!